src/style.css

```diff
@@ -115,7 +115,7 @@ body {
   box-shadow: 4px 3px 2px #000;
```

Comment:
- [OPITONAL] You did great styling your project. :+1: :+1: But I'll suggest you remove the red background color that flashes anytime a user moves the mouse into the app, its not subtle to the eye and makes looking at the UI a bit stain. :+1: :smile: